blog/2024-12-12-map/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: "Making sense of Kubernetes with Map view"
+description: "Introducing Map View feature to Headlamp"
```

Comment:
Making sense of Kubernetes with Map view -> Making sense of Kubernetes with Headlamp's Map view ?
I think Chris had also mentioned that the URLs should map the title, I am not sure how SEO-required this is, but if it is, we'd have to rename the folder.